src/apps/clients/data_info.clj

```diff
@@ -63,7 +63,8 @@
   (->> (slurp (get-file-contents user path))
        (string/split-lines)
        (remove empty?)
-       (drop 1)))
+       (drop 1)
+       (mapv string/trim)))
```

Comment:
This change isn't directly related to `CORE-1416`. This fixes a problem where trailing tabs in path list files were causing jobs to fail.